test/python/WMCore_t/MicroService_t/Tools_t/PycurlRucio_t.py

```diff
@@ -91,7 +93,7 @@ def testGetBlocksAndSizeRucio(self):
         self.assertTrue(len(resp) == 3)
         self.assertTrue(CONT2 in resp)
         self.assertTrue(len(resp[CONT4]) > 3)
-        self.assertItemsEqual(resp[CONT4][BLOCK].viewkeys(), ["blockSize", "locations"])
+        self.assertItemsEqual(resp[CONT4][BLOCK].keys(), ["blockSize", "locations"])
```

Comment:
I had the impression this wouldn't work on py3. Instead we should cast it to `list(my_dict)`, no?

Reply:
This is part of #10103, so i will mark it as resolved here. In any case, your suggestion is correct! Though, it is not broken in py3: `.keys()` in py3 is the same as `.viewkeys()` in py2. But if we use `.keys()` in both py2 and py3, it is going to be a list in py2 and a generator in py3. In this particular case it does not seem to be a problem since py2 is using a generator already, but it always nice to be consistent and have the same behavior in both py2 and py3.